packages/client/src/trackCCTX.ts

```diff
@@ -34,7 +34,7 @@ const fetchCCTXByInbound = async (
   try {
     const url = `${API}/zeta-chain/crosschain/inTxHashToCctx/${hash}`;
```

Comment:
is the url stil the same or should it change to inboundHashToCctx? 
mainly because before they it matched with the data key we have on line 37

Reply:
The URL is `/zeta-chain/crosschain/inTxHashToCctx/{inboundHash}`.

https://github.com/zeta-chain/node/blob/53883f575e8d7a6484584f3b400474e6aa610860/proto/zetachain/zetacore/crosschain/query.proto#L199